src/hosts/BitbucketAdapter.js

```diff
@@ -129,21 +100,43 @@ const BitbucketAdapter = (config: Config) => {
       return true;
     },
 
-    async getPullRequest(pullRequestId: string) {
+    async getPullRequest(pullRequestId: string): Promise<PullRequest> {
       const endpoint = `${apiBaseUrl}/pullrequests/${pullRequestId}`;
       const resp = await axios.get(endpoint, axiosGetConfig);
-      return resp.data;
+      const data = resp.data;
+      const approvals = data.participants
+        .filter(participant => participant.approved)
+        .map(participant => participant.user.username);
+
+      return {
```

Comment:
Now we don't expose implementation details about BB api's, we decide our own shape